modules/crud2.js

```diff
@@ -0,0 +1,51 @@
+// reset todo id
+export const idReset = (lists) => {
+  lists.forEach((todo, ind) => {
+    todo.id = ind + 1;
+  });
+};
+
+// logic to delete all completed todos
+export const deleteCompletedTodos = (todoListss, todoUl) => {
+  const newTodos = todoListss.filter((item) => item.completed !== true);
+  idReset(newTodos);
+  localStorage.setItem('todos', JSON.stringify(newTodos));
+
+  const elements = [...todoUl.children];
+  elements.forEach((ele) => {
+    if (ele.id === 'completed') {
+      ele.remove();
+    }
+  });
+};
+
+// functionality to mark a todo as completed
+export const selectTodo = (checkBox, todoTitle, item, itemContainer) => {
+  itemContainer.id = 'completed';
+  if (checkBox.checked) {
+    todoTitle.style.textDecoration = 'line-through';
+    item.completed = true;
+  } else {
+    todoTitle.style.textDecoration = 'none';
+    item.completed = false;
+  }
+};
```

Comment:
- [ ] [Optional] It will contain methods related to the status updates (completed: true/false). firstly the status will be `false` and then when the user clicks the checkbox then the status changes to `true` from false. please update it
### Error
![his](https://user-images.githubusercontent.com/81727988/214833734-f899eab7-a6fd-4690-997f-ebdb61e56282.jpg)